charts/pega/values.yaml

```diff
@@ -351,3 +353,31 @@ hazelcast:
   username: ""
   # Password to be used in client-server Hazelcast model for authentication
   password: ""
+
+# Stream (External Kafka) settings.
+# Disabled by default, you must not use stream tier when this setting is enabled.
+# Parameters trustStore,trustStorePassword,keyStore,keyStorePassword and jaasConfig are optional, their value depends on the securityProtocol.
+# For truststore/keystore certificate names, specify the Kubernetes secret that you have created to store SSL certificates for your deployment. For compatibility, see provider support for SSL certificate injection.
+# For more details please check "Support for providing credentials/certificates using External Secrets Operator".
+# Please note for stream add certificate names in 'global.certificatesSecrets','stream.trustStore', and 'stream.keyStore' as appropriate.
+# Pega does not support truststore and keystore certificates as plain text for stream.
+# By default, Pega supports only Java Key Store (.jks) format for truststore and keystore certificates.
+# Provide trustStorePassword, KeystorePassword, and jaasConfig values in Plain text. Alternatively, to secure the values leave values empty and configure them in the External Secrets Manager, make sure the keys in the secret are STREAM_TRUSTSTORE_PASSWORD, STREAM_KEYSTORE_PASSWORD and STREAM_JAAS_CONFIG respectively.
+stream:
+  enabled: false
```

Comment:
Should we enable it by default for the various yaml files if this is the recommended option? Or is this for backward compatibility until the next major version chart rev?

Reply:
This is for backward compatibility to minimize data loss.

Reply:
Do you plan to leave the embedded Kafka code in place for future releases?   What is the transition plan from embedded to external?

Reply:
As far as I know, code to support embedded Kafka will be removed in future Pega releases. We are going to publish steps to migrate from the embedded stream to external Kafka.